plugins/pulp_rpm/yum_plugin/util.py

```diff
@@ -241,13 +240,12 @@ def __init__(self, cmd=None, key_id=None, **kwargs):
         if not cmd:
             raise SignerError("Command not specified")
         self.cmd = cmd
-        self._cmdargs = shlex.split(cmd)
-        if not os.path.isfile(self._cmdargs[0]):
+        if not os.path.isfile(self.cmd[0]):
             raise SignerError(
-                "Command %s is not a file" % (self._cmdargs[0], ))
-        if not os.access(self._cmdargs[0], os.X_OK):
+                "Command %s is not a file" % (self.cmd[0], ))
+        if not os.access(self.cmd[0], os.X_OK):
             raise SignerError("Command %s is not executable" %
-                              (self._cmdargs[0], ))
+                              (self.cmd[0], ))
```

Comment:
On line 261 we need to change the if to:

if k.startswith('_') or k == 'cmd' or v is None:

and then probably a bunch of tests will need to have GPG_CMD removed.